### ISSUE TITLE
kanboard 1.0.29 - Internal Error: Identifier "language" is not defined

### ISSUE THREAD

**Issue by radimoto** [Busfreak/kanboard-color_filter#2]
Hi Busfreak,

I upgraded to Kanboard 1.0.29 today and unfortunately color_filter causes the following error:  

Internal Error: Identifier "language" is not defined  

If I move the plugin out of the way to /tmp/ Kanboard then runs OK.

**Comment by Busfreak** [comment]
Hi!

Try the update :smile:

**Comment by radimoto** [comment]
Thanks Busfreak, the color filters are now working again on my projects.  

One thing I notice though, is I used to be able to search across all projects but that now returns "Internal Error: Controller not found"

This is an example URL that would search all open tasks that are assigned the color red:  

www.example.com/kanboard/?controller=search&action=index&search=status%3Aopen+color%3A"red"

**EDIT**: just following up, I did some more testing and it looks like this issue isn't with Color_filter but is an issue with the search on the kanboard dashboard.  

If I use the search link, it does indeed work OK. The following examples works:  

https://www.example.com/kanboard/?controller=SearchController&action=index&search=status%3Aopen+and+color%3Ared  

I will submit open an issue with Kanboard.  

Thanks for updating Color_filter so quickly. It is really appreciated.
